log/noop_logger.go

```diff
@@ -0,0 +1,17 @@
+package log
+
+var _ Service = &NoOp{}
```

Comment:
I think the interface should be named `Logger` instead of `Service` no?

Reply:
changed